src/plots/mapbox/mapbox.js

```diff
@@ -747,17 +752,32 @@ proto.getView = function() {
     var mapCenter = map.getCenter();
     var center = { lon: mapCenter.lng, lat: mapCenter.lat };
 
+    var canvas = map.getCanvas();
+    var w = canvas.width;
+    var h = canvas.height;
+    var cUL = map.unproject([0, 0]).toArray();
+    var cUR = map.unproject([w, 0]).toArray();
+    var cLR = map.unproject([w, h]).toArray();
+    var cLL = map.unproject([0, h]).toArray();
```

Comment:
What about renaming `cLL` to `cBL` here?
Or you could simply use `c00`, `c01`, `c10`, c11` for the variable names.

Reply:
Or even better (IMO) is to simply to return without declaring extra variables and instead add inline comments
```js
    return {
        center: center,
        zoom: map.getZoom(),
        bearing: map.getBearing(),
        pitch: map.getPitch(),
        _derived: {
            coordinates: [
                map.unproject([0, 0]).toArray(), // top left
                map.unproject([w, 0]).toArray(), // top right
                map.unproject([w, h]).toArray(), // bottom right
                map.unproject([0, h]).toArray()  // bottom left
            ]
        }
    };
```

Reply:
Yeah, I agree. Inlining looks better than introducing awkward variable names in this case. Done in cacf1e9a8c6df92368b1b3c2d9f9f8a6f91445aa.